src/main/java/io/github/moulberry/notenoughupdates/listener/RenderListener.java

```diff
@@ -1685,4 +1696,24 @@ public void onGuiScreenKeyboard(GuiScreenEvent.KeyboardInputEvent.Pre event) {
 	public void onRenderLast(RenderWorldLastEvent event) {
 		CrystalMetalDetectorSolver.render(event.partialTicks);
 	}
+
+	/**
+	 * Support for switching between different pages in the RecipeView gui via right and left arrow key
+	 * @param event
+	 */
+	//Because GuiScreen.keyTyped does not fire the KEY_LEFT and KEY_RIGHT keys. Maybe some event cancelled it?
+	@SubscribeEvent
```

Comment:
i love making a 2000 lines file even longer with code relating to a feature that is entirely coded in a completely different class

Reply:
Is this comment still a valid change request? Should I find another class to put this method in? Or just create a new class entirely?